pkg/vm/opcode/opcode.go

```diff
@@ -188,9 +188,9 @@ const (
 	NUMEQUAL    Opcode = 0xB3
 	NUMNOTEQUAL Opcode = 0xB4
 	LT          Opcode = 0xB5
-	LTE         Opcode = 0xB6
+	LTE         Opcode = 0xB6 // LE
 	GT          Opcode = 0xB7
-	GTE         Opcode = 0xB8
+	GTE         Opcode = 0xB8 // GE
```

Comment:
Maybe we can rename them? Current names are probably just historic and we shouldn't have any issues renaming them.